coalib/settings/ConfigurationGathering.py

```diff
@@ -385,6 +386,21 @@ def gather_configuration(acquire_settings,
     local_bears, global_bears = fill_settings(sections,
                                               acquire_settings,
                                               log_printer)
+
+    arg_parser = default_arg_parser() if arg_parser is None else arg_parser
+
+    for arg_key, arg_value in sorted(
+            vars(arg_parser.parse_args(arg_list)).items()):
+        if arg_key == 'exclude_sections':
+            if arg_value is not None:
+                for x in arg_value:
+                    if x in sections:
+                        del sections[x]
+                    else:
+                        logging.warning(
+                            'The {} section doesn\'t exists'.format(x))
+                break
```

Comment:
it will iterate through all content of the list without it, we don't need args after the excluded sections.